server/seeder.js

```diff
@@ -11,29 +11,31 @@ const getRandomImageCount = function () {
 
 const truncateImageTable = function () {
   const initializeQuery = 'TRUNCATE TABLE images;';
-  databaseConnection.query(initializeQuery, (err, results) => {
-    if (err) {
-      throw (err);
-    } else {
-      console.log(results);
-    }
+  return new Promise((resolve, reject) => {
```

Comment:
If you want to use promises with MySQL in the future, you might want to look into Bluebird's `PromisifyAll` function. It saves you from needing to manually promisify callback-style functions. (I think this is OK as it is, though.)

Reply:
I will do that in the future